seacrowd/sea_datasets/cebuaner/cebuaner.py

```diff
@@ -0,0 +1,189 @@
+    LABEL_CLASSES = [
+        "O",
+        "B-PER",
+        "I-PER",
+        "B-ORG",
+        "I-ORG",
+        "B-LOC",
+        "I-LOC",
+        "B-OTHER",
+        "I-OTHER",
+    ]
+
+    # There are two annotators in the CebuaNER dataset but there's no canonical
+    # label. Here, we decided to create loaders for both annotators. The
+    # inter-annotator reliability is high so it's possible to treat either as
+    # gold-standard data.
+    dataset_names = sorted([f"{_DATASETNAME}_{annot}" for annot in _URLS.keys()])
+    BUILDER_CONFIGS = []
+    for name in dataset_names:
+        source_config = SEACrowdConfig(
+            name=f"{name}_source",
+            version=SOURCE_VERSION,
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=name,
+        )
+        BUILDER_CONFIGS.append(source_config)
+        seacrowd_config = SEACrowdConfig(
+            name=f"{name}_seacrowd_{SEACROWD_SCHEMA_NAME}",
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SEACrowd schema",
+            schema=f"seacrowd_{SEACROWD_SCHEMA_NAME}",
+            subset_id=name,
+        )
+        BUILDER_CONFIGS.append(seacrowd_config)
+
+    # Create a configuration that loads the annotations of the first annotator
+    # and treat that as the default.
+    BUILDER_CONFIGS.extend([
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_source",
+            version=SOURCE_VERSION,
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=_DATASETNAME,
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_seacrowd_{SEACROWD_SCHEMA_NAME}",
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SEACrowd schema",
+            schema=f"seacrowd_{SEACROWD_SCHEMA_NAME}",
+            subset_id=_DATASETNAME,
+        ),
+    ])
+
+    DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_1_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "tokens": datasets.Sequence(datasets.Value("string")),
+                    "ner_tags": datasets.Sequence(datasets.features.ClassLabel(names=self.LABEL_CLASSES)),
+                }
+            )
+        elif self.config.schema == f"seacrowd_{self.SEACROWD_SCHEMA_NAME}":
+            features = schemas.seq_label_features(self.LABEL_CLASSES)
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: DownloadManager) -> List[datasets.SplitGenerator]:
+        if self.config.subset_id == _DATASETNAME:
+            url = _URLS["annotator_0"] 
```

Comment:
can we avoid this hard-coded definition? perhaps making a var `_DEFAULT_ANNOTATOR` and giving brief comment string to it prior to class definition can be a better approach, wdyt?

Reply:
e.g state in comment "this dataset alignment between annotators is great and both has more-or-less same labels, hence defining this as first value on the index"

Reply:
Done!